src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1735,6 +1735,12 @@ public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstruct
             return null;
         }
 
+        public override BoundNode VisitOutDeconstructVarPendingInference(OutDeconstructVarPendingInference node)
```

Comment:
I think this should go into PreciseAbstractFlowPass and be sealed. Also, should we have the same override in the LocalRewriter?